### DISCUSSION TITLE
如何修改docker部署，网络模式network为host主机模式时，metric的端口改为非9091的默认端口，不采用bridge的网络模式

### DISCUSSION THREAD

**Discussion by xkz130** [milvus-io/milvus#22426] [Q&A and General discussion] [answered]
milvus集群模式，协调器和node的服务组件，监控指标的端口都是9092，但是目前使用主机模式的网络模式（使用bridge的网络模式可以映射为其他端口）时，9092的端口会有冲突，需要修改成其他的端口在网络模式是host网络的情况下，这个metric的端口可以如何修改，官网上没有看到相关配置方法

**Comment by LoveEachDay** [comment, marked as answer]
@xkz130 You can provide an env `METRICS_PORT=9091` for milvus components, use different ports for different components.
